packages/react/src/components/DetailsList/__snapshots__/DetailsList.test.tsx.snap

```diff
@@ -98,7 +98,7 @@ exports[`DetailsList handles paged updates to items within groups 1`] = `
                   min-width: 100%;
                 }
             data-automationid="GroupedList"
-            data-focuszone-id="FocusZone163"
+            data-focuszone-id="FocusZone161"
```

Comment:
A minor nit, but you may want to add `beforeEach` function which runs `resetIds`. This means each test will start at ID 0, mostly removing the need to rebase snapshots like this due to timing changes.

Reply:
FWIW; this `resetIds` pattern was never generalized. I'm not totally sure if that's something we could do, but once in a while I see updates like this in snapshots and suggest adding the resetId call and wonder if there is some way to pipe all the snapshots tests through some universal generalized setup that resets ids.